compiler-core/src/language_server/completer.rs

```diff
@@ -586,30 +596,28 @@ where
     }
 
     // Gets the position of the line with the first import statement in the file.
-    fn first_import_line_in_module(&'a self) -> Position {
-        let import_location = self
-            .module
-            .ast
-            .definitions
-            .iter()
-            .find_map(get_import)
-            .map_or(0, |i| i.location.start);
+    fn first_import_line_in_module(&'a self) -> (bool, Position) {
+        let import_location = self.module.ast.definitions.iter().find_map(get_import);
```

Comment:
Rather than iterate through all the definitions please just check if the first one is an import, and always place the import as the first thing in the module. Thank you.